acceptance/tests/cloud/fakeserver_client.go

```diff
@@ -0,0 +1,164 @@
+package cloud
+
+import (
+	"bytes"
+	"crypto/tls"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+)
+
+const (
+	// Validation are used in the validationBody to distinguish metrics for consul vs. collector during validation.
```

Comment:
we could also make the fakeserver handle different paths : /validation/consul vs. validation/collector, which might be clearer but was more work on the fakeserver.